pipeline_dp/dp_engine.py

```diff
@@ -38,7 +43,22 @@ def aggregate(self, col, params: AggregateParams,
       data_extractors: functions that extract needed pieces of information from
         elements of 'col'
     """
-
+    if params is None:
+      return None
+    self._report_generators.append(ReportGenerator(params))
+    self._add_report_stage(f"Clip values to {params.low, params.high}")
```

Comment:
No need to add these stages now, they will be added when the code is implemented.

Reply:
👍 Removed them and corresponding text based checks from the test (will just check for size of array).